services/src/handlers/layers.rs

```diff
@@ -455,3 +464,391 @@ async fn layer_handler<C: Context>(
 
     Ok(web::Json(collection))
 }
+
+#[derive(Debug, Deserialize, ToSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct AddLayerRequest {
+    pub layer: AddLayer,
+    #[schema(value_type = String, example = "05102bb3-a855-4a37-8a8a-30026a91fef1")]
+    pub collection_id: LayerCollectionId,
+}
+
+#[derive(Debug, Serialize, Deserialize, ToSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct AddLayerResponse {
```

Comment:
In the other handlers we use the generic `IdResponse<T>` struct as return type. Maybe use it here as well for uniformity?

Reply:
[ef718cf](https://github.com/geo-engine/geoengine/pull/676/commits/ef718cf9a8feb4115de82d1bf10f2f6992e39854)